solid/solid.js

```diff
@@ -1,275 +0,0 @@
-// You can find the basic Solid concepts explained in the Glossary.md file, inline comments talk about
```

Comment:
It seems like you have removed this file entirely, instead of moving it to `solid/solid-rest-api`. I guess that was a mistake?